### ISSUE TITLE
Export DB information to Excel file

### ISSUE THREAD

**Issue by calebdre** [sci-sim/sci-sim#10]
We need to have a script that parses the database and exports all the information to an excel and .csv file

**Comment by matthewbeckler** [comment]
From your short description it sounds like you are planning to make copies
of the database file. Could you instead make this code part of the web
interface?

**Comment by calebdre** [comment]
@matthewbeckler good point. We could have an endpoint that executes the script

**Comment by matthewbeckler** [comment]
Refresh my memory, are we still using a sqlite file as our backing
database, or does the deployed version use a web-scale database like
postgres? When we were on heroku we used postgres when deployed, and sqlite
when developing locally.

**Comment by calebdre** [comment]
We're still using sqlite both locally and on the deployed server. We haven't gotten around to switching it yet on the DO server for the deployed application. Ideally it shouldn't matter what database is being used for the script.

**Comment by matthewbeckler** [comment]
Right, but I was more worried about the concurrency support. Does sqlite
properly handle multiple users / multiple concurrent reads/writes without
data corruption or excessive performance degradation? I'm sure it's just
fine for now, but if you ever notice it being slow, that may be one thing.

**Comment by calebdre** [comment]
This is very true. SQLite doesn't support concurrency..I want to switch to MySQL later on
